chia/cmds/configure.py

```diff
@@ -92,6 +93,13 @@ def configure(
             config["full_node"]["target_peer_count"] = int(set_peer_count)
             print("Target peer count updated")
             change_made = True
+        if enable_data_server:
+            config["data_layer"]["run_server"] = str2bool(enable_data_server)
```

Comment:
Are you guaranteed to have the `data_layer` yaml node at this point?

Reply:
Yeah, there's that hazard.  I'm on the fence about fixing it ad-hoc here.  It's both more of a hazard with data layer being new and also an existing hazard for everything here I think.  Anyways, https://github.com/Chia-Network/chia-blockchain/pull/13199.

Reply:
Or perhaps https://github.com/Chia-Network/chia-blockchain/pull/13220 instead.